AmethystAPI/src/minecraft/src-client/common/client/gui/gui/UIControl.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+class UIControl {
+public:
+  BUILD_ACCESS(this, std::string, layerName, 0x18);
```

Comment:
This should be replaced with an actual field in the struct, see: https://github.com/FrederoxDev/Amethyst/pull/39#issuecomment-1895228515